### PR TITLE
Show a warning when bcWalletTest is skipped

### PR DESCRIPTION
**Pull request by pirapira**

### CODE REVIEW

File: test/tools/jsontests/BlockChainTestsBoost.cpp [hunk 1]
```diff
@@ -58,7 +58,10 @@ class bcTestFixture {
 
 		//skip wallet test as it takes too much time (250 blocks) run it with --all flag
 		if (casename == "bcWalletTest" && !test::Options::get().wallet)
+		{
+			clog << "skipping bcWalletTest. Use --all to run it.\n"
```
Comment (winsvega) [inline test/tools/jsontests/BlockChainTestsBoost.cpp]: why only bcWallet test? there are plenty of other tests which are skipped.

Comment (pirapira) [inline test/tools/jsontests/BlockChainTestsBoost.cpp]: This code is reached only when `bcWalletTest` is specified.

**Comment by pirapira** [comment]
The test needs to be updated.

**Comment by winsvega** [comment]
build failed for some reason

**Comment by pirapira** [comment]
Anyway I need to rebase.

**Comment by pirapira** [comment]
I'll combine this with #4404

**Pull request closed without merging**